src/masternode-payments.h

```diff
@@ -6,6 +6,7 @@
 #ifndef MASTERNODE_PAYMENTS_H
 #define MASTERNODE_PAYMENTS_H
 
+#include "util.h"
```

Comment:
`#include "util.h"` is already there in masternode-payments.cpp, if you need DBG somewhere else where masternode-payments.h is included imo it's better to include util.h there explicitly rather than having chaining dependencies through masternode-payments.h.